java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -156,6 +156,16 @@ public interface CommonProperty {
             .validationPredicate(Utils::isNonNullNonEmptyString)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty METRICS_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.metrics.concurrency.reserved")
+            .description("The reserved concurrency for the table metrics lambda.\n " +
+                    "See reserved concurrency overview at: \r\n https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html\r\n")
```

Comment:
The new properties have carriage return characters, some extra spaces around the new line characters, and extra new line characters at the end of the description.

We haven't used carriage returns in any of the other properties, is there a reason to have them here?

Please can we remove the extra spaces, and the extra new line characters at the end?

Reply:
Addressed